### PR TITLE
UML-44 build containers in pipeline

### PR DESCRIPTION
**Pull request by andrewpearce-digital**

### CODE REVIEW

**Comment by andrewpearce-digital** [comment]
This pipeline is working. However the app itself returns a 200 and an error. Fixing that can be a separate tiket as it's just a scaffold of an app at present.

**Comment by TomTucka** [comment]
I'm Happy with the changes, I would definitely squash the commits though

File: docker-compose.yml [hunk 1]
```diff
@@ -17,6 +18,7 @@ services:
 
   viewer-app:
     container_name: viewer-app
+    image: 311462405659.dkr.ecr.eu-west-1.amazonaws.com/use_my_lpa/view_lpa_front:latest
```
Comment (nsmithuk) [inline docker-compose.yml]: @andrewpearce-digital Is it easy to change this to `viewer` also?
And what are you thoughts on using `use_an_lpa` everywhere now? Other than the github repo perhaps?

Comment (andrewpearce-digital) [reply docker-compose.yml]: It's straightforward, naming of github and ecr managed from opg-org-infra.

I think changing the name of an ECR destroys the old and creates a new one, losing the images.
Changing the repo name is an edit.

I want to make another ticket after we have a follow up about naming things, then rename the things.

**Pull request merged**